fission/src/ui/panels/GraphicsSettingsPanel.tsx

```diff
@@ -0,0 +1,184 @@
+                            }}
+                            step={1}
+                        />
+                        <Slider
+                            min={MIN_SHADOW_MAP_SIZE}
+                            max={World.SceneRenderer.renderer.capabilities.maxTextureSize}
+                            value={shadowMapSize}
+                            label="Shadow Map Size"
+                            onChange={(_, value: number | number[]) => {
+                                setShadowMapSize(value as number)
+                                World.SceneRenderer.changeCSMSettings({
+                                    shadowMapSize: value as number,
+                                    maxFar: maxFar,
+                                    lightIntensity: lightIntensity,
+                                    fancyShadows: fancyShadows,
+                                    cascades: cascades,
+                                    antiAliasing: antiAliasing,
+                                })
+                            }}
+                            step={1024}
+                        />
+                        {Spacer(10)}
+                        <Box alignSelf={"center"}>
+                            <Button
+                                value="Reset Default"
+                                onClick={() => {
+                                    setShadowMapSize(4096)
+                                    setMaxFar(30)
+                                    setLightIntensity(5)
+                                    setCascades(4)
+
+                                    World.SceneRenderer.changeCSMSettings({
+                                        shadowMapSize: 4096,
+                                        maxFar: 30,
+                                        lightIntensity: 5,
+                                        fancyShadows: fancyShadows,
+                                        cascades: 4,
+                                        antiAliasing: antiAliasing,
+                                    })
+                                }}
+                            />
+                        </Box>
+                    </>
+                ) : (
+                    <></>
+                )}
+                <div className="flex items-center justify-center mt-1 mb-0.5 mx-[5%]">
+                    <SectionLabel size={LabelSize.Medium} className="text-center">
+                        Hot-reload Settings
+                    </SectionLabel>
+                </div>
+                <SectionDivider />
+                <Checkbox
+                    label="Anti-Aliasing"
+                    defaultState={antiAliasing}
+                    onClick={checked => {
+                        // saving the new preference
+                        setAntiAliasing(checked)
+                        setReload(true)
+                    }}
+                    tooltipText="Requires a browser refresh"
+                />
```

Comment:
Should probably have some sort of user confirmation here before you auto refresh and reset the current scene. Could do this by adding some sort of confirmation after the user clicks accept with a change to this option or could be saved with some sort of "Refresh is required to apply some settings" message.